app/services/av_characterizer_service.rb

```diff
@@ -128,10 +128,12 @@ def extract_part(track, part_type)
 
   # @param [String] time a date with format like 'UTC 2020-02-27 06:06:04' - although sometimes UTC doesn't come through
   def to_iso_time(time)
+    time.gsub!(':', '-') # ensure all colons are dashes to standardize data
```

Comment:
I don't think you've made it very clear that this handles the special case where the date is formated YYYY:MM:DD, maybe you can add that to the documentation.  Also worth considering, why replace one delimiter with another?  Can you just remove the delimiter altogether?

Reply:
yeah, I could do that

Reply:
stripped out the colons and dashes instead of replacing one with the other

bonus: the regex loos like a cool emoji face